lib/db.js

```diff
@@ -721,6 +758,16 @@ module.exports = function (
     return this.pool.post('/account/' + account.uid.toString('hex') + '/verifyEmail')
   }
 
+  DB.prototype.verifyToken = function (tokenVerificationId, token) {
```

Comment:
Yeah, maybe its my fault for calling the argument `token` in the database PR, which I did to try and follow convention over there. But it is inaccurate, sorry if it misled you.

Maybe it doesn't belong in the payload at all, seeing as it's just the one field? We could pass it in the path instead if that's less misleading. Or we could keep it in the payload and I could just give it a better name than `token` over there?

But aside from that, I agree that the method here should just take `uid` as its second argument and unbuffer `{ uid: uid }` instead.